src/pages/rekey-reason/rekey-reason.ts

```diff
@@ -175,6 +189,40 @@ export class RekeyReasonPage {
     }
   }
 
+  formIsValid() {
+    const rekeyReasonProvided = this.formGroup.get('ipadIssue').value ||
+      (this.formGroup.get('otherSelected').value);
+
+    const reasonForRekeyIsValid = !this.formGroup.get('otherSelected').value ||
+      (this.formGroup.get('otherSelected').value
+        && this.formGroup.get('otherReasonUpdated').valid);
+
+    if (rekeyReasonProvided && reasonForRekeyIsValid) {
+      return true;
+    }
+
+    if (!rekeyReasonProvided) {
+      this.createToast('Provide at least one reason for rekey');
```

Comment:
Don't think we need to use a toast message. Toast is used on the office page but that's because the errors can scroll off the screen. This page is small so I think we should just use the standard validation (e.g. same as comms page or WRTC page)